manifests/0000_30_openshift-apiserver-operator_07_deployment.yaml

```diff
@@ -43,6 +43,8 @@ spec:
         env:
         - name: IMAGE
           value: quay.io/openshift/origin-openshift-apiserver:v4.2
+        - name: OPERATOR_IMAGE
+          value: docker.io/openshift/origin-cluster-openshift-apiserver-operator:v4.0
```

Comment:
this does not seem right?

Reply:
that's taken from image-references, I can fix it in both places but not only one of them. What would the correct value be? `s/docker/quay/`?

Reply:
fixed